packages/framer-motion/src/animation/interfaces/motion-value.ts

```diff
@@ -118,6 +118,7 @@ export const animateMotionValue = (
          * Animate via WAAPI if possible.
          */
         if (
+            !transition.isHandoff &&
```

Comment:
Let's add a comment here to explain why we want to animate individual transforms when it's a handoff one.